client/state/plugins/marketplace/actions.ts

```diff
@@ -2,17 +2,45 @@
  * Internal dependencies
```

Comment:
This file has the key change with most of the important where async business logic is implemented as async action creators.

Reply:
Tests will be written for this in a follow up PR